scripts/zones/Batallia_Downs/mobs/Stalking_Sapling.lua

```diff
@@ -1,27 +1,17 @@
 -----------------------------------
 -- Area: Batallia Downs
 --  MOB: Stalking Sapling
+-- Note: PH for Tottering Toby
 -----------------------------------
 require("scripts/zones/Batallia_Downs/MobIDs");
 require("scripts/globals/fieldsofvalor");
+require("scripts/globals/mobs");
 
 function onMobDeath(mob, player, isKiller)
     checkRegime(player,mob,72,1);
     checkRegime(player,mob,73,1);
 end;
 
 function onMobDespawn(mob)
-    local mobID = mob:getID();
-
-    if (TOTTERING_TOBY_PH[mobID] ~= nil) then
-        local ToD = GetServerVariable("[POP]Tottering_Toby");
-        if (ToD <= os.time() and not GetMobByID(TOTTERING_TOBY):isSpawned()) then
-            if (math.random(1,20) == 5) then
-                UpdateNMSpawnPoint(TOTTERING_TOBY);
-                GetMobByID(TOTTERING_TOBY):setRespawnTime(GetMobRespawnTime(mobID));
-                SetServerVariable("[PH]Tottering_Toby", mobID);
-                DisallowRespawn(mobID, true);
-            end
-        end
-    end
+    phOnDespawn(mob,TOTTERING_TOBY_PH,20,3600); -- 1 hour
```

Comment:
Toby can take way longer than an hr, so either the spawn rate is much lower or the cooldown can be much longer.

Reply:
I was using the information from [here](http://ffxiclopedia.wikia.com/wiki/Talk:Tottering_Toby) primarily.  It seems that when people camp the correct placeholder and kill it asap, toby's cooldown is assumed to be about an hour.

Reply:
which would indicate he's lower rate. I had to do him for several diff magian weapons on retail when those were new. we were murdering every sapling zone wide, wasn't a single one left alive in map. unfortunately theres now ay to get handle one what the rate really should be (I suspect SE has tiers for it they group NMs into) so I guess the real question is..Do you really want to go with 1 in 20 chance there? I guess Ima merge it for the same reason I changed my mind about caring if blubbery is 100% accurate, and anyone objecting can PR the changes. No real reason to matter when magian trials aren't done yet